galite-tests/src/test/kotlin/org/kopi/galite/demo/client/ClientForm.kt

```diff
@@ -46,7 +46,7 @@ class ClientForm : ReportSelectionForm() {
           label = "DynamicReport",
           help = " Create Dynamic Report",
   ) {
-    key = Key.F8          // key is optional here
+    key = Key.F6          // key is optional here
```

Comment:
undo

Reply:
F8 already exists (for createReport) , F6 is for dynamicReport